docs/index/secondary-index-guide.md

```diff
@@ -118,9 +118,11 @@ also load data to the SI table with the same number of segments as the main tabl
 the main table will also load data to the SI table.
 
  **NOTE**:
- * In case of data load failure to the SI table, then we make the SI table disable by setting a hive serde
- property. The subsequent main table load will load the old failed loads along with current load and
- makes the SI table enable and available for query.
+ * In case of data load failure to the SI table, segments of main table and SI table will not be in sync.
```

Comment:
can we give one example for this scenario for easy understanding ? 
ex:
segment 1 loaded successfully to maintable and SI
segment 2 loaded successfully to maintable and SI
segment 3 failed to load. 
show segment differences between maintable and SI table.
so in this case for segment 1 and segment 2 pruned by SI and segment 3 filter with help of maintable.

Reply:
As discussed already examples present in the design doc which present in CARBONDATA-4162.